config.json

```diff
@@ -20,87 +20,258 @@
+      "deprecated": false
+    },
+    {
+      "slug": "isogram",
+      "uuid": "d1a98c79-d3cc-4035-baab-0e334d2b6a57",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 3,
       "topics": [
         "algorithms",
         "conditionals",
-        "filtering",
-        "logic",
         "loops",
         "strings"
       ]
     },
     {
-      "slug": "rna-transcription",
-      "uuid": "ca7c5ef1-5135-4fb4-8e68-669ef0f2a51a",
-      "core": false,
+      "slug": "scrabble-score",
+      "uuid": "d081446b-f26b-41a2-ab7f-dd7f6736ecfe",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 3,
+      "topics": [
+        "games",
+        "loops",
+        "maps",
+        "strings"
+      ]
+    },
+    {
+      "slug": "kindergarten-garden",
+      "uuid": "42a2916c-ef03-44ac-b6d8-7eda375352c2",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "arrays",
+        "classes",
+        "optional_values",
+        "variables"
+      ]
+    },
+    {
+      "slug": "grade-school",
+      "uuid": "aadde1a8-ed7a-4242-bfc0-6dddfd382cf3",
+      "core": true,
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
         "conditionals",
+        "filtering",
+        "integers",
+        "lists",
+        "sorting",
+        "strings"
+      ]
+    },
+    {
+      "slug": "luhn",
+      "uuid": "34dde040-672e-472f-bf2e-b87b6f9933c0",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 4,
+      "topics": [
+        "algorithms",
+        "conditionals",
+        "loops",
+        "pattern_matching",
+        "security"
+      ]
+    },
+    {
+      "slug": "clock",
+      "uuid": "459fda78-851e-4bb0-a416-953528f46bd7",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 4,
+      "topics": [
+        "classes",
         "logic",
+        "text_formatting",
+        "time"
+      ]
+    },
+    {
+      "slug": "markdown",
+      "uuid": "88610b9a-6d3e-4924-a092-6d2f907ed4e2",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 3,
+      "topics": [
+        "refactoring",
+        "text_formatting"
+      ]
+    },
+    {
+      "slug": "twelve-days",
+      "uuid": "d41238ce-359c-4a9a-81ea-ca5d2c4bb50d",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 5,
+      "topics": [
+        "lists",
+        "strings",
+        "text_formatting"
+      ]
+    },
+    {
+      "slug": "tournament",
+      "uuid": "49377a3f-38ba-4d61-b94c-a54cfc9034d0",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 5,
+      "topics": [
+        "conditionals",
         "loops",
         "maps",
-        "strings",
-        "transforming"
+        "parsing"
       ]
     },
     {
-      "slug": "isbn-verifier",
-      "uuid": "7961c852-c87a-44b0-b152-efea3ac8555c",
+      "slug": "rest-api",
+      "uuid": "2abe6eed-c7e4-4ad7-91e3-778bc5176726",
+      "core": true,
```

Comment:
`rest-api` still has [some sizable issues](); it should probably be a side exercise for now. It might merit promotion when it is more developed.